### PR TITLE
[ticket/14365] Add core event to the function topic_review()

### PR DESCRIPTION
**Pull request by rxu**
Add core event to the function topic_review() in includes/functions_posting.php
to allow modifying posts list to review.

<a href="https://tracker.phpbb.com/browse/PHPBB3-14365">PHPBB3-14365</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set Event

File: phpBB/includes/functions_posting.php [hunk 1]
```diff
@@ -1129,6 +1129,32 @@ function topic_review($topic_id, $forum_id, $mode = 'topic_review', $cur_post_id
...
 
+	/**
+	* Event to modify the posts list for topic reviews
+	*
+	* @event core.topic_review_modify_post_list
+	* @var	array	attachments			Array with the post attachments data
+	* @var	int		cur_post_id			Post offset ID
+	* @var	int		forum_id			The topic's forum ID
+	* @var	string	mode				The topic review mode
+	* @var	array	post_list			Aray with the post IDs
```
Comment (marc1706) [inline phpBB/includes/functions_posting.php]: Should be Array and not Aray

**Comment by marc1706** [comment]
Rest looks good.

**Comment by rxu** [comment]
@marc1706 updated #4085.

**Comment by marc1706** [comment]
This unfortunately didn't make it into 3.1.8. Would you be so kind and update the version to 3.1.9-RC1? Thank you.

**Comment by rxu** [comment]
Updated #4085.

**Pull request merged**